[GLOSSARY] change (#430)
zkSNACKs/WasabiDoc: docs/glossary/Glossary.md

Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
:::

:::details
### Change

> When in a spending transaction the provided input coins are larger than the value send to the destination address, then the leftover change is send back to an address of the same wallet.

Reviewer: When in a spending transaction the provided input coins are larger than the value sent to the destination address, then the leftover change is sent back to an address of the same wallet.

Author: ```suggestion
When in a spending transaction the provided input coins are larger than the value sent to the destination address, then the leftover change is sent back to an address of the same wallet.
```

> When in a spending transaction the provided input coins are larger than the value sent to the destination address, then the leftover change is sent back to an address of the same wallet.

Reviewer 2: ```suggestion
In a spending transaction where the provided input coins are larger than the value sent to the destination address, then the leftover change is sent to a new address of the same wallet.
```

> CoinJoins create standardized equal value outputs, and the change of a CoinJoin can easily be linked to the input, thus they have 0 anonymity set.

Reviewer 2: I think this sentence should be removed from this definition.

Author: I was thinking the same... 
```suggestion
```
